opentok/opentok.py

```diff
@@ -62,9 +72,9 @@ class OpenTok(object):
     def __init__(self, api_key, api_secret, api_url='https://api.opentok.com', timeout=None):
         self.api_key = str(api_key)
         self.api_secret = api_secret
-        self.api_url = api_url
         self.timeout = timeout
         self._proxies = None
+        self.endpoint = Endpoint(api_url, self.api_key)
```

Comment:
Since there are multiple endpoints and the class name should be `endpoints`, I think it's best to call it `endpoints`.